### DISCUSSION TITLE
Match file text with variable

### DISCUSSION THREAD

**Discussion by Nat-intosh** [WarmUpTill/SceneSwitcher#647] [Q&A] [answered]
Hello, to automate my OBS setup i have a file text with the current hour generated by Snaz, and i'm wondering if this is possible to have a variable or another text file in the "match" condition of the "content of local file".
It's quite difficult to explain that but here's a little plan of what i would like to do 

![Capture d’écran 2022-12-29 140258](https://user-images.githubusercontent.com/54398543/209954976-42a90e6f-8bcc-41a6-9d94-175c92898783.png)

I want to put a variable or another text file where the arrow is located.
If this is possible, how can i do that ?
Thank's in advance
- Nat

**Comment by WarmUpTill** [comment, marked as answer]
That is not possible in current releases unfortunately.

Not sure if it helps for your use case but you could however try this build:
https://github.com/WarmUpTill/SceneSwitcher/actions/runs/3794812200

It will allow you to use variables you have defined elsewhere in the plugin in text boxes using the following pattern:
`${variableNameHere}`

For example, the action in the below screenshot would result in sending the following message:
`Some other text A some more text`

![VarTest](https://user-images.githubusercontent.com/19472752/210140665-1f84951f-f5f6-4b11-ae08-5be991d56212.PNG)

Let me know if you have any questions or run into issues!
